.storybook/main.ts

```diff
@@ -0,0 +1,24 @@
+/** @type { import('@storybook/svelte-vite').StorybookConfig } */
```

Comment:
is there any reason these `.storybook/*` files could go either in `.config` or in their own package in `js/storybook`.

We could have a packages with a `config` and `app` subfolder, which solves the problem of where the `Introduction.mdx` should live as well as keeps a folder out of the root (we have so many and these files will modified infrequently). It looks like you pass a path to the config to the storybook CLI: https://storybook.js.org/docs/react/api/cli-options#build